changelog_unreleased/blog-post-intro.md

```diff
@@ -0,0 +1,11 @@
+---
+author: "Sosuke Suzuki"
+authorURL: "https://github.com/sosukesuzuki"
+title: "Prettier 2.4: new bracketSameLine option and TypeScript 4.4 support!"
+---
+
+This release adds new `bracketSameLine` option! And `jsxBracketSameLine` is deprecated.`bracketSameLine` is generic version of the `jsxBracketSameLine` option that works for HTML, JSX, Vue, and Angular.
```

Comment:
Not 100% happy with “which” in this suggestion but I can’t think of anything better at the moment.

```suggestion
This release renames the `jsxBracketSameLine` option to `bracketSameLine`, which supports HTML, Vue, and Angular in addition to JSX. The old name has been deprecated.
```